presto-benchmark/src/test/java/com/facebook/presto/benchmark/BenchmarkDecimalAggregation.java

```diff
@@ -66,10 +67,18 @@ public final MemoryLocalQueryRunner getQueryRunner()
         @Setup
         public void setUp()
         {
+            queryRunner = new MemoryLocalQueryRunner();
             queryRunner.execute(format(
                     "CREATE TABLE memory.default.orders AS SELECT orderstatus, cast(totalprice as %s) totalprice FROM tpch.sf1.orders",
                     type));
         }
+
+        @TearDown
+        public void tearDown()
```

Comment:
This commit seems unneeded as benchmarks are one-time run only and are not part of Travis automation

Reply:
Some benchmarks do tear down. I don't see why this one shouldn't.